### ISSUE TITLE
Why are "Exchange messages with programs other than Firefox" and "Control browser proxy settings" required for the update?

### ISSUE THREAD

**Issue by braiam** [mozilla/contain-facebook#816]
It makes no sense that given the extension functionality to do this, there is? I've read the [list of possible permissions asked](https://support.mozilla.org/en-US/kb/permission-request-messages-firefox-extensions), and none of them seems to address any usability of the extension. I reviewed the manifest.json and can't see where those permissions are declared to gain knowledge about it. Did someone compromise the addons account? Or it's firefox misinterpreting the manifest.json file and a bug in Firefox instead?

**Comment by aplaice** [comment]
Are you sure that the messages came up for the "Facebook Container" extension and not the "Firefox Multi-Account Containers" one?

Looking at the XPIs from AMO for both extensions, the latter does ask for the `nativeMessaging` and `proxy` permissions in its `manifest.json`, while the former does not. Looking at the source code for ["Facebook Container"](https://github.com/mozilla/contain-facebook/blob/master/src/manifest.json#L45) and for ["Multi-Account Containers"](https://github.com/mozilla/multi-account-containers/blob/main/src/manifest.json#L12-L27), that's also confirmed.

If you are sure, then it is indeed extremely worrying! (A bug in Firefox or AMO!)

<hr/>

("Firefox Multi-Account Containers" is now apparently starting to ask for this permission in order to allow integration with Mozilla VPN — AFAIU to allow specific containers to be configured to send their traffic via the VPN.)

**Comment by braiam** [comment]
Yeah, it seems that I misread the box (which only appeared once, and not after I went down the rabbit hole) so closing this issue.
